massa-protocol-worker-2/src/connectivity.rs

```diff
@@ -130,46 +150,67 @@ pub fn start_connectivity_thread(
                 ));
             }
 
+            println!("Protocol started");
```

Comment:
I think this println (and some others) can be removed now

Reply:
Removed bunch of them